api/base/serializers.py

```diff
@@ -282,7 +281,15 @@ def lookup_attribute(self, obj, lookup_field):
         bracket_check = _tpl(lookup_field)
         if bracket_check:
             source_attrs = bracket_check.split('.')
-            return get_nested_attributes(obj, source_attrs)
+            # If you are using a nested attribute for lookup, and you get the attribute wrong, you will not get an
+            # error message, you will just not see that field. This allows us to have slightly more dynamic use of
+            # nested attributes in relationship fields.
+            try:
+                return_val = get_nested_attributes(obj, source_attrs)
+            except KeyError:
```

Comment:
Maybe, but not in the situation that I was having trouble with it. What are you thinking of?